backend/baseclass.pm

```diff
@@ -36,6 +36,7 @@ use OpenQA::NamedIOSelect;
 use constant FULL_SCREEN_SEARCH_FREQUENCY => $ENV{OS_AUTOINST_FULL_SCREEN_SEARCH_FREQUENCY} // 5;
 use constant FULL_UPDATE_REQUEST_FREQUENCY => $ENV{OS_AUTOINST_FULL_UPDATE_REQUEST_FREQUENCY} // 5;
 use constant DEFAULT_FFMPEG_CMD => 'ffmpeg -y -hide_banner -nostats -r 24 -f image2pipe -vcodec ppm -i - -pix_fmt yuv420p';
+use constant SSH_SERIAL_READ_BUFFER_SIZE => 4096;
```

Comment:
Would that make sense as a ENV variable?

Reply:
No. I don't think it's even necessary to overwrite this. If it will become necessary then we can still change it